src/StackAdmin/Azs.AzureBridge.Admin/test/Get-AzsAzureBridgeActivation.Tests.ps1

```diff
@@ -0,0 +1,56 @@
+$TestRecordingFile = Join-Path $PSScriptRoot 'Get-AzsAzureBridgeActivation.Recording.json'
```

Comment:
we dont really need a new file for each test, pls feel free to group the tests in to a single file, if it makes sense

Reply:
I could group the Product related tests in one file but, as validation goes, I found that it was easier to keep tests for each cmdlet separate